src/models/syncRule.ts

```diff
@@ -42,6 +48,14 @@ export default class SyncRule {
         return this.syncRule.builder.metadataIds;
     }
 
+    public get useDefaultIncludeExclude(): boolean {
+        return this.syncRule.builder.useDefaultIncludeExclude;
+    }
+
+    public get metadataExcludeIncludeRules(): MetadataIncludeExcludeRules | undefined {
```

Comment:
Since we have ``useDefaultIncludeExclude``, it would make sense to disallow it being undefined.

Note: Existing rules will lead to an error when not finding it. We usually add ``?? {}`` for this particular case.

Reply:
When useDefaultIncludeExclude is equal to true the sync rule does not contains include exclude rules to avoid duplicate and innecessary rules stored in dhis2 store and to avoid bugs if default rules changes on the future.

During the synchronization, the rules will be hardcoded in models if MetadataIncludeExcludeRules is undefined

Reply:
done